packages/vector/src/services/pgvector-provider.ts

```
@@ -185,95 +204,106 @@ export class PgVectorProvider implements VectorProvider {
+	}
+
+	private async generateEmbeddings(texts: string[]): Promise<number[][]> {
+		const response = await this.openai.embeddings.create({
+			model: "text-embedding-ada-002",
+			input: texts,
+		});
+
+		return response.data.map((item) => item.embedding);
+	}
+
+	private formatResults(rows: any[]): QueryResult {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)